### PR TITLE
Add apiperf to packages and prepare for 2.3 release

### PR DESCRIPTION
**Pull request by pakmarkthub**
This PR:
- adds apiperf to the deb and rpm packages.
- updates changelogs.
- bumps up version to 2.3 and resets debian version back to 1.

**Note:** PR #195 must be merged before merging this PR.

### CODE REVIEW

**Comment by drossetti** [comment]
do we need to reset rpm version to 1 too?

**Comment by drossetti** [comment]
can you rebase to later master?

**Comment by pakmarkthub** [comment]
> do we need to reset rpm version to 1 too?

The rpm version is already 1 from the start.

> can you rebase to later master?

Done.

File: packages/gdrcopy.spec [hunk 1]
```diff
@@ -288,7 +289,12 @@ rm -rf $RPM_BUILD_DIR/%{name}-%{version}
 
 
 %changelog
-* Mon Feb 01 2021 Pak Markthub <pmarkthub@nvidia.com> %{GDR_VERSION}-%{_release}
+* Fri Jul 23 2021 Pak Markthub <pmarkthub@nvidia.com> %{GDR_VERSION}-%{_release}
+- Removed automatically-generated build id links..
+- Removed gdrcopy-kmod from the Requires field..
+- Added apiperf test.
+- Various updates in README.
```
Comment (drossetti) [inline packages/gdrcopy.spec:296]: do you need to add "Revamp gdrdrv to fix race-condition bugs." here?

File: src/gdrdrv/gdrdrv.h [hunk 2]
```diff
@@ -27,7 +27,7 @@
 #define GDRDRV_TOSTRING(s)            GDRDRV_STRINGIFY(s)
 
 #define GDRDRV_MAJOR_VERSION    2
-#define GDRDRV_MINOR_VERSION    2
+#define GDRDRV_MINOR_VERSION    3
```
Comment (drossetti) [inline src/gdrdrv/gdrdrv.h:30]: has the kernel ABI changed at all?

Comment (drossetti) [reply src/gdrdrv/gdrdrv.h:30]: never mind, the implementation has changed enough that we want to store this version in the module itself

**Review by drossetti — APPROVED** [review]

**Comment by pakmarkthub** [comment]
Thank you Davide. Merging the PR.

**Pull request merged**